docs/users.md

```diff
@@ -0,0 +1,86 @@
+# Create a Database User #
+
+You can create a MongoDB database user to authenticate to your MongoDB resource using SCRAM. First, [create a Kubernetes secret](#create-a-user-secret) for the new user's password. Then, [modify and apply the MongoDB CRD](#modify-the-mongodb-crd).
+
+You cannot disable SCRAM authentication.
+
+## Create a User Secret
+
+1. Copy the following example secret.
+
+     ```
+     ---
+     apiVersion: v1
+     kind: Secret
+     metadata:
+       name: <db-user-secret>  # corresponds to spec.users.passwordSecretRef.name in the MongoDB CRD
+     type: Opaque
+     stringData:
+       password: <my-plain-text-password> # corresponds to spec.users.passwordSecretRef.key in the MongoDB CRD
+     ...
+     ```
+1. Update `metadata.name` with the name of the secret and `stringData.password` with the user's password.
```

Comment:
Offer clearer guidance here that you can provide anything you want for  <db-user-secret>. I think users might ask what the name of the secret is without realizing they can provide any value here.